### PR TITLE
Fixes some general content

### PR DESCRIPTION
**Pull request by gabrieluizramos**
Fixes some wording typos, some accents and terms for better understanding.

### CODE REVIEW

File: admission/03-prework/10-guided-exercises/README.pt-BR.md [hunk 1]
```diff
@@ -15,9 +15,9 @@
 
 ## Enunciados
 
-Agora que você já conhece certos conceitos básicos da programação, é hora de
-você ver na prática as coisas que você consegue criar. Não é um teste , mas é
-super importante você praticar!! Os projetos a criar são:
+Agora que você já conhece certos conceitos básicos da programação, é hora de ver na prática as coisas que você consegue criar. Não é um teste, mas é super importante você praticar!!
```
Comment (vdemario) [inline admission/03-prework/10-guided-exercises/README.pt-BR.md:18]: I'm not sure if this is documented somewhere but it looks like there's an 80 column limit on the markdown files, we should split this line.

File: admission/03-prework/12-exercises/01-coin-convert/README.pt-BR.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # coinConvert
 
-Neste exercício vamos nos concentrar em atribuir _valores_ em _variáveis_. O
+Neste exercício vamos nos concentrar em atribuir _valores_ à _variáveis_. O
```
Comment (vdemario) [inline admission/03-prework/12-exercises/01-coin-convert/README.pt-BR.md:3]: This `à` should be an `a`.

File: admission/03-prework/12-exercises/02-restaurant-bill/README.pt-BR.md [hunk 3]
```diff
@@ -1,15 +1,11 @@
 # Conta do restaurante
 
-Imagine que você saiu para comer com as suas quatro melhores amigas. O valor
-total da conta foi de 50 reais, lembrando que você deve adicionar os 10% de
-imposta. Vocês querem dividir a conta igualmente entre as cinco. E para isso
-você vai criar um programa.
+Imagine que você saiu para comer com as suas quatro melhores amigas e o valor total da conta foi de 50 reais (lembrando que você deve adicionar os 10% da taxa de serviço). Vocês querem dividir a conta igualmente entre as cinco e você vai criar um programa para realizar essa tarefa.
```
Comment (vdemario) [inline admission/03-prework/12-exercises/02-restaurant-bill/README.pt-BR.md:3]: 80 column limit here too.

**Comment by raulingg** [comment]
@vdemario Thanks for the review.

@gabrieluizramos let me know when you have the minor corrections about mdlink fixed

File: admission/03-prework/10-guided-exercises/README.pt-BR.md (hunk 1 above)
Comment (gabrieluizramos) [reply admission/03-prework/10-guided-exercises/README.pt-BR.md:18]: Totally! Just realized that I messed up almost all the lines that I've touched 😅 Pushing the fixes right now.

**Comment by gabrieluizramos** [comment]
@vdemario @raulingg thank you guys for the review! Just pushed the changes 🚀

File: admission/03-prework/10-guided-exercises/README.pt-BR.md (hunk 1 above)
Comment (vdemario) [reply admission/03-prework/10-guided-exercises/README.pt-BR.md]: It's documented btw, I noticed now: https://github.com/Laboratoria/admission-curriculum#testing.

Comment (gabrieluizramos) [reply admission/03-prework/10-guided-exercises/README.pt-BR.md]: Yeah! Thank you! My bad!

**Pull request merged**